tests/common/dualtor/dual_tor_utils.py

```diff
@@ -1382,10 +1375,11 @@ def update_linkmgrd_probe_interval(duthosts, tbinfo, probe_interval_ms):
     if 'dualtor' not in tbinfo['topo']['name']:
         return
 
-    logger.info("Increase linkmgrd probe interval on {} to {}ms".format(duthosts, probe_interval_ms))
+    logger.info("Update linkmgrd probe interval on {} to {}ms".format(duthosts, probe_interval_ms))
     cmds = []
-    cmds.append('sonic-db-cli CONFIG_DB HSET "MUX_LINKMGR|LINK_PROBER" "interval_v4" "{}"'.format(probe_interval_ms))
-    duthosts.shell_cmds(cmds=cmds)
+    cmds.append('sonic-db-cli CONFIG_DB HSET "MUX_LINKMGR|LINK_PROBER" "interval_v4" "{}"'
+                .format(probe_interval_ms))
+    duthosts.shell(cmds=cmds)
```

Comment:
Ansible `shell` module expects argument `cmd`, not `cmds`. And `cmd` need to be a string, not a list.

Reply:
Fixed